src/backend/management/commands/run_telegram_bot.py

```diff
@@ -99,66 +106,110 @@ async def show_menu(update: Update, context: ContextTypes.DEFAULT_TYPE):
     else:
-        current_order[user.id] = {empanada_type: quantity}
-
-    logger.info(f"{user.first_name} ({user.id}) added {quantity} {empanada_type}")
-    await update.message.reply_text(
-        f"Agregué {quantity} de {empanada_type} al pedido grupal!"
-        "\n\nSi querés seguir agregando empanadas podes volver a pedir con /pedido_individual."
-        "\nSi nadie mas quiere agregar pedidos, pueden finalizar el pedido desde el grupo con /finalizar_pedido."
+        current_order[user.id] = {product_name: quantity}
+
+    logger.info(f"{user.first_name} ({user.id}) added {quantity} {product_name}")
+    await context.bot.edit_message_text(
+        text=f"Agregué {quantity} {product_name.lower()} al pedido grupal!"
+             "\n\nSi querés seguir agregando productos podes volver a pedir con /pedido_individual."
+             "\nSi nadie mas quiere agregar productos, pueden finalizar el pedido desde el grupo con /finalizar_pedido.",
+        chat_id=query.message.chat_id,
+        message_id=query.message.message_id
     )
 
     return ConversationHandler.END
 
+
+async def start_message(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    await update.message.reply_text("Bienvenido a PediGroup, para registrar un pedido debes usar el comando /pedido_individual.")
+    return ConversationHandler.END
+
+
 def start_bot():
     application = ApplicationBuilder().token(settings.TELEGRAM_BOT_TOKEN).build()
     
     application.add_handler(CommandHandler("iniciar_pedido", start_order))
     application.add_handler(CommandHandler("finalizar_pedido", finish_order))
 
     individual_order_handler = ConversationHandler(
-        entry_points=[CommandHandler("pedido_individual", show_menu)],
+        entry_points=[CommandHandler("pedido_individual", show_menu),
+                      MessageHandler(filters.ALL, start_message)],
         states={
             TYPE_SELECTION: [CallbackQueryHandler(handle_type_selection)],
-            QUANTITY: [MessageHandler(filters.TEXT & ~filters.COMMAND, handle_quantity)],
+            QUANTITY: [CallbackQueryHandler(handle_quantity)],
         },
         fallbacks=[],
     )
 
     application.add_handler(individual_order_handler)
 
     application.run_polling()
+
+
+def registrar_grupo_y_usuario_si_debe(group, user):
+    if not Group.objects.filter(id_app__contains=group.id).exists():
+        Group.objects.create(name=group.title, id_app=group.id)
+    registrar_usuario_si_debe(user)
+
+
+def registrar_usuario_si_debe(user):
+    if not User.objects.filter(id_app__contains=user.id).exists():
+        User.objects.create(first_name=user.first_name, last_name=user.last_name, 
+                            username=user.username, id_app=user.id)
+
+
+def registrar_pedido_de_usuario(product_name, quantity, user):
+    pedigroup_user = User.objects.get(id_app__contains=user.id)
+    pedigroup_product = Product.objects.get(name__contains=product_name)
```

Comment:
si ya sabemos que van a ser exactamente esos id y product_name que queremos buscar no es necesario el contains
```suggestion
    pedigroup_user = User.objects.get(id_app=user.id)
    pedigroup_product = Product.objects.get(name=product_name)
```